frontend/src/components/browser/inspectorBrowser.js

```diff
@@ -189,13 +194,161 @@ class InspectorBrowser extends Component {
     );
   }
 
+  /* 
+  Fetch selected clip based on the chosen clip.id.
+  */
+  fetchSelectedClip(clipId) {
+    if (this.props.inspector.id == undefined) {
+      return "";
+    }
+
+    if (!this.props.clips[clipId]) {
+      console.warn("Id ", clipID, " does match with a clip");
+      return "";
+    } else {
+      return this.props.clips[clipId];
+    }
+  }
+
+  /**
+   * @param {Object[int, list]} dict A list of objects. 
+   * @param {int} objectId The id for the asked object.
+   * @param {string} key The key for which value should be returned. 
+
+   Fetch a specific value based on the key. First finds the matching object
+   with the objectId. Afterwords returning the value of the key. 
+   */
+  fetchObjectInDict(dict, objectId, key) {
+    if (dict != undefined && objectId != undefined && key != undefined) {
+      let res = "";
+      Object.values(dict).map((obj) => {
+        if (Object.is(obj.id, objectId)) {
+          console.log("obj: ", obj);
+          res = obj;
+        }
+      });
+      return res[key];
+    } else {
+      return "";
+    }
+  }
+
   /* Render the clip mode displayed in inspector */
   renderClip() {
-    return (
-      <div>
-        <h3 className={styles.browserInspectorHeader}>Clip</h3>
-      </div>
-    );
+    if (this.props.inspector != undefined && this.props.clips != undefined) {
+      let clip = this.fetchSelectedClip(this.props.inspector.id);
+      console.log("clip: ", clip);
+      console.log("filter: ", this.props.filter);
+      return (
+        <div>
+          {/* Displays heading for the clip mode and selected clip*/}
+          <p className={styles.browserInspectorHeader}>Clip</p>
+          <p className={styles.browserInspectorHeader}>{clip.name}</p>
+          {/* Displays info of the seleceted clip*/}
+          <Table striped bordered hover size="sm">
+            <thead>
+              <tr>
+                <th>Category</th>
+                <th>Info</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>Camera</td>
+                <td>
+                  {this.fetchObjectInDict(
+                    this.props.cameras,
+                    clip.camera,
+                    "name"
+                  )}
+                </td>
+              </tr>
+              <tr>
+                <td>Folder</td>
+                <td>
+                  {this.fetchObjectInDict(
+                    this.props.folders,
+                    clip.folder,
+                    "absolutePath"
```

Comment:
Show relative path instead of absolute. The folder has a function getPath which can be used.